### PR TITLE
Update mimetypes

### PR DESCRIPTION
**Pull request by juancarlospaco**
- Update `mimetypes` module `const mimes*`.
- Only the `const` changed, the rest of the code is untouched for easy merge.

Mimes BEFORE = **460**
Mimes AFTER  = **1864**
Improvement  = **1404**

Mime Sources:
- [Apache Foundation](http://svn.apache.org/viewvc/httpd/httpd/trunk/docs/conf/mime.types?view=markup)
- [IANA](http://www.iana.org/assignments/media-types/)

### CODE REVIEW

File: lib/pure/mimetypes.nim [hunk 1]
```diff
@@ -14,481 +14,1862 @@ type
...
-    "fm": "application/x-maker",
-    "fb": "application/x-maker",
-    "book": "application/x-maker",
-    "fbdoc": "application/x-maker",
-    "mif": "application/x-mif",
-    "wmd": "application/x-ms-wmd",
-    "wmz": "application/x-ms-wmz",
-    "com": "application/x-msdos-program",
-    "exe": "application/x-msdos-program",
-    "bat": "application/x-msdos-program",
```
Comment (Araq) [inline lib/pure/mimetypes.nim]: Now it's "text/plain"?

Comment (juancarlospaco) [reply lib/pure/mimetypes.nim]: Done :+1:

Comment (Araq) [reply lib/pure/mimetypes.nim]: Well yes, but now I'm skeptical about your approach. Can't the be generated from some official spec?

**Comment by juancarlospaco** [comment]
Question:
As it is now, code fails if passing UPPER cased extensions (`"exe"` vs `"EXE"`), 
I think is best to do the compare with lowercased `string` on the `proc` of this module, 
is Ok if I send another PR to add `toLowerAscii` to the `string` argument before the compare :grey_question:

File: lib/pure/mimetypes.nim (hunk 1 above)
Comment (juancarlospaco) [reply lib/pure/mimetypes.nim]: They are from the linked source, nothing shady on the code.
I generate it from official spec, they are not made up.

But the official-ity of the official is also kinda relative, along history extensions have been reused, etc.
The same happens across other langs too, Ive been checking Rust, NodeJS, Python, Go, and their Mimes are not exactly the same, they are a lot different.

The point is to keep the well known extensions mapped to its mimes.

Comment (Araq) [reply lib/pure/mimetypes.nim]: Ok.

**Comment by Araq** [comment]
Yes.

**Pull request merged**